GitOut/Features/Wpf/AttachedProperties/TextZoom.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Windows;
+using System.Windows.Input;
+using System.Windows.Media;
+
+namespace GitOut.Features.Wpf.AttachedProperties
+{
+    public class TextZoom
+    {
+        private static readonly double Scale = .2;
```

Comment:
For later; this could also be set from persisted storage. Should this be called ScaleIncrement instead?

Reply:
It's used to decrement as well though.

Reply:
ScaleStep?